DependencyInjection/Configuration.php

```diff
@@ -42,6 +42,12 @@ public function getConfigTreeBuilder()
                 ->end()
                 ->scalarNode('token_ttl')
                     ->defaultValue(3600)
+                    ->validate()
+                        ->ifTrue(function ($ttl) {
+                            return !is_numeric($ttl);
```

Comment:
also why not simply make it an `integerNode` as I suspect that floats do not really make sense here ..